src/Compiler/pars.fsy

```diff
@@ -2375,8 +2375,23 @@ objectImplementationMember:
      { let rangeStart = rhs parseState 1
        $3 $1 $2 rangeStart }
 
-  | opt_attributes staticMemberOrMemberOrOverride error
-      { [] }
+  | opt_attributes staticMemberOrMemberOrOverride recover
```

Comment:
Hey @auduchinok, if you have a minute please take a glance if this looks good to you.

Reply:
Overall it would be better to have the parser changes separately, but if this proves to be enough here, it can go together I think. Just a small ask for the future.

Reply:
This change is to make override completion can be triggered in interface implementations or object expressions

Reply:
That I understand, I meant having it separately as a pre-PR for this PR, since it also just improves the error recovery :)
But again, not a big deal here.